job-service-unit-tests/src/test/java/com/github/jobservice/core/api/TestUtil.java

```diff
@@ -36,6 +36,8 @@ public static void setSystemEnvironmentFields(Map<String, String> newenv) throws
                 map.putAll(newenv);
             }
         }
+        System.out.println(env);
+        System.out.println(System.getenv("JOB_SERVICE_DATABASE_PASSWORD"));
```

Comment:
Was leaving these lines in intentional?

Reply:
Um, looks like I put this to debug some tests. Its unintentional actually.